zk.go

```diff
@@ -32,17 +32,19 @@ type ZK struct {
 	loc       *time.Location
 	lastData  []byte
 	disabled  bool
+	forceUDP  bool
 	capturing chan bool
 }
 
-func NewZK(host string, port int, pin int, timezone string) *ZK {
+func NewZK(host string, port int, pin int, timezone string, forceUDP bool) *ZK {
```

Comment:
@willylambert Instead of using forceUDP, please change to `network string`. 
This network value only have 2 values: `tcp` and `udp`.
You can create an enum to hold them.